README.md

```diff
@@ -1 +1,51 @@
-# Portfolio-Website
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Linters Set-Up
+
+> Linters are used for keeping a code that is in sync with best code practices
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used

```

Comment:
- [ ] You need to add technologies that you have used in this project. :+1: You can check out this [readme](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/README.md) to get a good understanding. :blush: